src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/compat/patchouli/PatchouliCompat.java

```diff
@@ -258,7 +259,7 @@ public void init(FMLCommonSetupEvent event) {
                 '4', goldInlayNorthWest,
                 '0', api.strictBlockMatcher(Blocks.IRON_BARS),
                 'I', api.strictBlockMatcher(Blocks.IRON_BARS),
-                'R', api.strictBlockMatcher(Blocks.LIGHTNING_ROD)
+                'R', api.propertyMatcher(Blocks.LIGHTNING_ROD.defaultBlockState(), LightningRodBlock.FACING, LightningRodBlock.WATERLOGGED)
```

Comment:
Shouldn't this one NOT be waterlogged?

Reply:
that is what it does the defined properties are the ones that are checked against the state (others like powered are ignored)

Reply:
Ah, I see.